src/app/space/add-space-overlay/add-space-overlay.component.less

```diff
@@ -8,6 +8,7 @@
   bottom: 0;
   left: 0;
   background-color: rgba(0, 0, 0, .8);
+  overflow-y: auto;
```

Comment:
@christianvogt with this, you are essentially allowing for a double page scroll. I understand why, but I just wanted to let you know that in case you have another idea. When I make my browser small enough, I can scroll the overlay and, when that has finished scrolling, the page then scrolls.

Reply:
@mindreeper2420 ya I'm aware of the double scroll. I was thinking of applying a class to the body to set `overflow: hidden` however then I found `add-app-overlay.component.less` was already setting `overflow-y: auto` and to fix one would also then require applying the fix there as well.

We should look at creating a single common component for displaying such overlays that take care of all of these quirks instead of one off fixes. Isn't that what patternfly is suppose to do for us?

I think the double scroll is the lesser evil here until such time.

Reply:
Agreed that there should be single common component to handle these quirks. Regarding PatternFly, this does not actually exisit in PatternFly and is purely an OSIO component, so we have complete control over it (though it can always be submitted to PF for inclusion).

Agreed that this is the lesser of two evils for the time being, but also that we should standardize this interaction.